### DISCUSSION TITLE
Cannot sign up to GitHub Talent Community

### DISCUSSION THREAD

**Discussion by DeleMike** [community/community#136281] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi, 

I tried to join GitHub's Talent Community but I kept receiving this error message _"It looks like we are having trouble processing your submission. Please check for any validation errors and try again"_. I have attached a screenshot too.

How can I resolve this and also join the Talent Community?

Thank you.
![Screenshot 2024-08-19 at 11 55 16](https://github.com/user-attachments/assets/ad5f298b-3acc-4494-9302-4d838f6cd81f)

**Comment by pierrelouisguichard** [comment, marked as answer]
Hi,

I encountered the same issue with a 422 Unprocessable Content error. After investigating, I found that the problem was due to an invalid location value. To resolve it, I left the City and State/Province fields empty and entered NA for the Postal/Zip Code.

Hope this helps!

**Comment by DeleMike** [comment]
Oh, it worked! Thank you, Pierre Guichard!
